contracts/teller-network/Escrow.sol

```diff
@@ -143,6 +143,7 @@ contract Escrow is Pausable, MessageSigned {
 
         EscrowTransaction storage trx = transactions[_escrowId];
 
+        require(trx.paid == false, "Transaction already paid");
```

Comment:
I am curious, because you have a boolean, is it possible to do:
```
!trx.paid
```
instead of comparing to a boolean?

Reply:
(same question for the 2 belows)

Reply:
It is possible. I'll make the change since it improves readability!